### ISSUE TITLE
4E4D0826 - DB: Raging Blast 2

### ISSUE THREAD

**Issue by Momoske** [xenia-project/game-compatibility#1162]
[Marketplace] https://marketplace.xbox.com/en-US/Product/DB-Raging-Blast-2/66acd000-77fe-1000-9115-d8024e4d0826

Tested on both https://github.com/xenia-project/xenia/commit/d3d12 AND https://github.com/xenia-project/xenia/commit/master

# Issues:
Game won't recognize any input past the loading cinematic - Pressing START (as asked) won't do anything
The start button works fine at the start of the game to accelerate the sponsorships though.

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3051715/xenia.zip)

# Screenshot(s):
http://prntscr.com/n8pa63 (input works up until this screen)

# Labels:
state-intro

**Comment by ZolaKluke** [comment]
Duplicate of: https://github.com/xenia-project/game-compatibility/issues/51
please remove

**Comment by Ulquiorr4-Cifer** [comment]
@ZolaKluke apparently that one is demo and this one full game, will remove if i'm wrong

**Comment by Momoske** [comment]
Yep, that's exactly what I wanted to reply. This is the full game ripped from my Xbox CD game, and DB: Raging Blast 1 works perfectly. It's the same issue as #51 but not the exact same game :/ I can still remove if needed.

**Comment by surenzxx** [comment]
Any update on this issue?

**Comment by StardustGogeta** [comment]
As of 7-28-2020, this game seems to work perfectly for me on D3D12. I've tried several missions in the Galaxy Mode and Battle Zone game modes and have experienced no issues.

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/e7cd2ffffa418002ea5e526a3edba734d5f24990

Watched a 5v5 CPU vs CPU match. 50-267FPS with vsync off and shaders cached.

![dbz](https://user-images.githubusercontent.com/31154606/103384895-bd4ac380-4abd-11eb-9968-7d41d670bba6.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5755920/xenia.zip)
[xenia.z01.zip](https://github.com/xenia-project/game-compatibility/files/5755921/xenia.z01.zip)

state-gameplay | tech-middleware-criware

**Comment by Ghaleon7771** [comment]
Works almost perfectly on my old PC (I3 3240, R7 360, 8GB RAM) with some FPS drops in the city stage or when i go really far away from the opponent in a stage with too much models/textures on it, but definitely playable. Downloaded a 100% save data to have all the characters already unlocked too.

**Comment by FofoX11** [comment]
how do i downloud this game please help i dont see anywhere i can downlouyd it

**Comment by joebog1** [comment]
It works perfectly minus a strange bug I encountered while playing. The "You must sign in to save data. Sign in?" Dialogue page when opening the game is now in a state where hitting "Yes" causes the dialogue page to loop and ask for Sign in again, this loops forever, regardless of opening or closing the game. Not sure why this would happen now as my save is not new, perhaps I accidentally closed while saving at some point and the data is now corrupted in some manner. Do you have any suggestions on how to fix/debug this issue?

**Comment by Goonslaper** [comment]
this is literally the most playable xenia game I've tried so far.

**Comment by Al-AmeenAdewunmi** [comment]
Music doesn't work for some reason.
